build.ps1

```diff
@@ -11,6 +11,8 @@ param (
     [string]$NuGetPFXPath,
     [switch]$SkipXProj,
     [switch]$SkipCSProj,
```

Comment:
Makes sense to me. It only removes two command line scenarios (described above). Will remove it.